### PR TITLE
Rename "SUSE Customer Center Configuration" to "Product Registration" (FATE#318800)

### PR DESCRIPTION
**Pull request by mvidner**
[FATE#318800](https://fate.suse.com/318800)

This is only a proposal, wait with merging until PM confirm the name.

### CODE REVIEW

File: src/clients/registration.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+scc.rb
```
Comment (lslezak) [inline src/clients/registration.rb]: Wow, a nice trick how to redirect to a different client...

**Comment by lslezak** [comment]
LGTM, so let's wait for the PM's feedback...

**Comment by mvidner** [comment]
@jsrain re-renamed

**Comment by jsrain** [comment]
LGTM, provided the pending check passes

**Pull request merged**